### PR TITLE
Add inertiajs

### PR DESCRIPTION
**Pull request by mulligaj**
## Description
Install interiaJS via composer/ modified middleware files as instructed so that it works as intended.
[Inertia Server-side Setup](https://inertiajs.com/server-side-setup)

Also modified how files are generated via:
```
php artisan module:make
```
It will by default still use traditional MVC, but with a minor change in the generated Controller it can be switched to use Inertia instead.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -50,7 +51,8 @@
         "psr-4": {
             "App\\": "app/",
             "Database\\Factories\\": "database/factories/",
-            "Database\\Seeders\\": "database/seeders/"
+            "Database\\Seeders\\": "database/seeders/",
+            "Modules\\": "app/Modules"
```
Comment (zooley) [inline composer.json]: Why was this added?

Comment (mulligaj) [reply composer.json]: I was testing things and forgot to remove it. I force pushed an update that removes this.
